src/hangar/merger.py

```diff
@@ -12,14 +12,14 @@
 -------------
 
 In the current implementation, only fast-forward and a very simple three-way merge
-algorithm are implemented. All user facing API calls should be funnled through the
+algorithm are implemented. All user facing API calls should be funneled through the
 :function:`select_merge_algorithm` function
 
 .. note::
 
     In the current implementation, it is not possible to stop a merge in progress or
     to revert a bad merge commit. All revert like operations should be made by
-    creating new branchs from the last "good" state, after which new merge
+    creating new branches from the last "good" state, after which new merge
```

Comment:
In order to make this behavior "gentler", couldn't we switch to a service branch and then rename atomically once the merge is done?